definitions/procedures/pulp/remove.rb

```diff
@@ -0,0 +1,89 @@
+module Procedures::Pulp
+  class Remove < ForemanMaintain::Procedure
+    metadata do
+      description 'Remove pulp2'
+      for_feature :pulp2
```

Comment:
lets remove this 'for_feature' here, it seems to hide this procedure once pulp2 is removed, but a user may want to re-run the command after a ctrl-c for example, after pulp-server has been removed, but before the directory removals have happened